dvc/command/imp.py

```diff
@@ -6,7 +13,13 @@
 class CmdImport(CmdBase):
     def run(self):
         try:
-            self.project.imp(self.args.url, self.args.out)
```

Comment:
It should probably be either ntpath or posixpath, depending on the type of remote. ntpath for windows local path and posixpath for everything else.

Reply:
@efiop, I don't understand how this affects the output, I'm only getting the last part of a remote URL and using it as the output. I'm assuming that it won't contain any weird character, as it follows the URI scheme.

For example:
```
dvc import remote://myremote/filename -> filename
```

I don't know if I'm missing something.

Reply:
Hm, you are right. I thought that ntpath is not smart enough to extract base name even from posix paths, but I was clearly wrong.